### PR TITLE
Patch the init container with compute resources

### PR DESCRIPTION
**Pull request by feizhe1996**
Fix for https://github.com/k8tz/k8tz/issues/83.
Previously, the compute resources specified in values.yaml did not get passed down to the initContainer specs. When the namespace is guarded by a resource quota, each container needs to specify the resource requirement, otherwise, k8s throws quota errors.

Since the resources definition is a multi-line string block, the fix tries to pass a json formatted string as an argument to the cli.

### CODE REVIEW

**Comment by feizhe1996** [comment]
cc: @yonatankahana for review

**Review by yonatankahana — CHANGES_REQUESTED** [review]

**Comment by yonatankahana** [comment]
ping @feizhe1996

**Comment by feizhe1996** [comment]
> ping @feizhe1996

Ack

**Comment by feizhe1996** [comment]
Ping @yonatankahana  for review

**Comment by feizhe1996** [comment]
ping @yonatankahana for review

**Comment by jiriksykora** [comment]
we are still waiting for mr

**Comment by feizhe1996** [comment]
> we are still waiting for mr

ping @yonatankahana  again

File: charts/k8tz/values.yaml [hunk 1]
```diff
@@ -11,6 +11,8 @@ injectedInitContainerName: k8tz
 injectAll: true
 cronJobTimeZone: false  # requires kubernetes >=1.24.0-beta.0 with 'CronJobTimeZone' feature gate enabled (alpha)
 verbose: false
+initContainerResources: {} # resources claimed for the bootstrap init container
+resources: {} # resources claimed for the k8tz controller
```
Comment (yonatankahana) [inline charts/k8tz/values.yaml:15]: resources already exist in `values.yaml`

File: charts/k8tz/values.yaml [hunk 2]
```diff
@@ -11,6 +11,8 @@ injectedInitContainerName: k8tz
 injectAll: true
 cronJobTimeZone: false  # requires kubernetes >=1.24.0-beta.0 with 'CronJobTimeZone' feature gate enabled (alpha)
 verbose: false
+initContainerResources: {} # resources claimed for the bootstrap init container
```
Comment (yonatankahana) [inline charts/k8tz/values.yaml:14]: please document the new value in the chart README.md

**Review by yonatankahana — CHANGES_REQUESTED** [review]
hi, sorry for my late responses. please fix this small things and let's release it this weekend.

**Comment by feizhe1996** [comment]
Commit updated cc: @yonatankahana

**Pull request merged**